### ISSUE TITLE
One Machine For Cuckoo & Another Machine For Analysis

### ISSUE THREAD

**Issue by seantree** [cuckoosandbox/cuckoo#1919]
Hi Guys,

I have one linux machine running cuckoo on it and I have another Windows Machine Which running 7 VM's, now the thing is I am not getting how to do routing for this I have read the documentation.
.
Kindly help me on this issue.
.
Waiting for some positive reply.
.
Thanks & Regards
Seantree

**Comment by doomedraven** [comment]
DO Searxh in issues, that way of cuckoo setup is wrong

**Comment by seantree** [comment]
Thanks @doomedraven  for the reply can't it be possible if both the physical machines running on Linux Machines and 1 Linux machine having cuckoo running while another linux machine running Mutliple VM's??
.
Waiting for some positive reply.
.
Thanks & Regards
Seantree

**Comment by doomedraven** [comment]
No, that should be the same host, as physical can be only one host per host but you wants vms

**Comment by seantree** [comment]
Thanks @doomedraven for your reply.

**Comment by doomedraven** [comment]
You are welcome :)
